gallery/src/components/NxProgressBar/NxProgressBar.tsx

```diff
@@ -0,0 +1,190 @@
+        This component does not support indeterminate state.
+      </NxP>
+      <NxTable>
+        <NxTable.Head>
+          <NxTable.Row>
+            <NxTable.Cell>Prop</NxTable.Cell>
+            <NxTable.Cell>Type</NxTable.Cell>
+            <NxTable.Cell>Required</NxTable.Cell>
+            <NxTable.Cell>Details</NxTable.Cell>
+          </NxTable.Row>
+        </NxTable.Head>
+        <NxTable.Body>
+          <NxTable.Row>
+            <NxTable.Cell><NxCode>value</NxCode></NxTable.Cell>
+            <NxTable.Cell>0 &ge; number &le; <NxCode>max</NxCode></NxTable.Cell>
+            <NxTable.Cell>Yes</NxTable.Cell>
+            <NxTable.Cell>Value indicating progress. A number from  0 to <NxCode>max</NxCode>.</NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell><NxCode>max</NxCode></NxTable.Cell>
+            <NxTable.Cell>number &gt; 0 and number &ge; <NxCode>value</NxCode></NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>
+              By default this is set to 100.
+            </NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell><NxCode>label</NxCode></NxTable.Cell>
+            <NxTable.Cell>
+              string
+            </NxTable.Cell>
+            <NxTable.Cell>yes</NxTable.Cell>
+            <NxTable.Cell>
+              <NxP>
+                Default label under the progress bar.
+                If <NxCode>labelSuccess</NxCode> is not specified, this label will remain
+                shown but with the success style when progress reaches 100%.
+              </NxP>
+              <NxP>
+                When the <NxCode>variant</NxCode> is "small" or "inline" or{' '}
+                <NxCode>inlineCounter</NxCode> is set to true, the label will not be displayed.
+                Instead, it will be assigned as an aria-label to the progress element.
+              </NxP>
+              <NxP>
+                This is a required property for accessibility.
+              </NxP>
+            </NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell><NxCode>labelSuccess</NxCode></NxTable.Cell>
+            <NxTable.Cell>
+              string
+            </NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>
+              When specified and progress reaches 100%,
+              this string will be displayed instead of <NxCode>label</NxCode>.
+            </NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell><NxCode>labelError</NxCode></NxTable.Cell>
+            <NxTable.Cell>
+              string
+            </NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>
+              When specified, error styling is applied, the component progress is set to 0%, and,
+              this string will be displayed instead of <NxCode>label</NxCode>.
```

Comment:
```suggestion
              When specified, error styling is applied, the component progress is set to 0%, and
              this string will be displayed instead of <NxCode>label</NxCode>.
```